src/Presenters/PackagesPresenter.php

```diff
@@ -88,7 +105,7 @@ public function count()
     /**
      * Get an iterator for the packages.
      *
-     * @return \ArrayIterator
+     * @return \ArrayIterator|Package[]
```

Comment:
This type signature is incorrect I'm afraid. The thing returned is not an array.